components/PrivateRoute.tsx

```diff
@@ -0,0 +1,22 @@
+import router, { useRouter } from "next/router";
+import { useContext } from "react"
+import { magic } from "../lib/Magic/magic";
+import { UserContext } from "../lib/userContext"
+
+
+
```

Comment:
In general, I think a single empty line is best. Only adding this comment here, but applies to all files with multiple empty lines in a row.